### PR TITLE
Pt 166332175 slash test case

### PR DESCRIPTION
**Pull request by uwiger**
See [PT #166332175](https://www.pivotaltracker.com/story/show/166332175)

Slash test case doesn't yet complete successfully, though the basic flow seems to execute correctly.

### CODE REVIEW

File: apps/aechannel/src/aesc_fsm.erl [hunk 1]
```diff
@@ -1765,10 +1768,12 @@ handle_common_event_(cast, {?CHANNEL_CLOSING, Info} = Msg, _St, _, D) ->
     D1 = log(rcv, ?CHANNEL_CLOSING, Msg, D),
     case check_closing_event(Info, D1) of
         {can_slash, Round, SignedTx} ->
-            {ok, SlashTx, Updates} = slash_tx_for_signing(Round, SignedTx, D1),
+            %% {ok, SlashTx, Updates} = slash_tx_for_signing(Round, SignedTx, D1),
```
Comment (lucafavatella) [inline apps/aechannel/src/aesc_fsm.erl]: Why not deleting this section?

File: apps/aehttp/src/sc_ws_api_jsonrpc.erl [hunk 2]
```diff
@@ -528,12 +544,16 @@ process_request(#{<<"method">> := <<"channels.close_solo">>}, FsmPid) ->
     no_reply;
 process_request(#{<<"method">> := <<"channels.slash">>}, FsmPid) ->
     lager:debug("Channel WS: slash message received"),
-    aesc_fsm:slash(FsmPid),
-    no_reply;
+    case aesc_fsm:slash(FsmPid) of
+        ok -> no_reply;
+        {error, _Reason} = Err -> Err
```
Comment (lucafavatella) [inline apps/aehttp/src/sc_ws_api_jsonrpc.erl]: Worth a line in release notes? See proposal at #2503

File: apps/aehttp/src/sc_ws_api_jsonrpc.erl [hunk 3]
```diff
@@ -528,12 +544,16 @@ process_request(#{<<"method">> := <<"channels.close_solo">>}, FsmPid) ->
...
+    end;
 process_request(#{<<"method">> := <<"channels.settle">>}, FsmPid) ->
     lager:debug("Channel WS: settle message received"),
-    aesc_fsm:settle(FsmPid),
-    no_reply;
+    case aesc_fsm:settle(FsmPid) of
+        ok -> no_reply;
+        {error, _Reason} = Err -> Err
```
Comment (lucafavatella) [inline apps/aehttp/src/sc_ws_api_jsonrpc.erl]: Worth a line in release notes? See proposal at #2503

File: apps/aechannel/src/aesc_fsm.erl (hunk 1 above)
Comment (uwiger) [inline apps/aechannel/src/aesc_fsm.erl]:
```suggestion
```

**Pull request merged**